portal/models/intervention_strategies.py

```diff
@@ -231,15 +233,15 @@ def update_user_card_html(intervention, user):
                 classification='indefinite'))
 
         def thank_you_block(name, registry):
-            greeting = _(u"Thank you, {}.").format(name)
+            greeting = _("Thank you, {}.").format(name)
```

Comment:
Can you change this to use printf-like syntax? FYI The variable name (`name`) gives translators extra info about how to translate the message:

```
_("Thank you, %(full_name)s.", full_name=name)
```

Reply:
On it